docs/integrations/destinations/redshift.md

```diff
@@ -94,12 +94,26 @@ connection only. S3 is secured through public HTTPS access only.
 5. (Optional)
    [Create](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) a
    staging S3 bucket \(for the COPY strategy\).
-6. Create a user with at least create table permissions for the schema. If the schema does not exist
-   you need to add permissions for that, too. Something like this:
 
-```
-GRANT CREATE ON DATABASE database_name TO airflow_user; -- add create schema permission
-GRANT usage, create on schema my_schema TO airflow_user; -- add create table permission
+### Permissions in Redshift
+Airbyte writes data into two schemas, whichever schema you want your data to land in, e.g. `my_schema`
```

Comment:
probably should clarify that this is specific to dv2 since that hasn't gone GA yet

Reply:
🤷 I think I'd rather just let this go into the GA release, or I'd be ok with the astute reader giving us slightly too many permissions that sets them up for success migrating to v2